ts/parts-3d/Column.ts

```diff
@@ -218,10 +218,11 @@ seriesTypes.column.prototype.translate3dShapes = function (): void {
                 [{
                     x: (tooltipPos as any)[0],
                     y: (tooltipPos as any)[1],
```

Comment:
You can try to use `PositionObject` or `TooltipPositionerPointObject` instead of `any`

Reply:
Thanks! I will test it later today!

Reply:
We use `Highcharts.Point.tooltipPos?` Which is an optional parameter and that is the reason of the problem.

Reply:
I would leave it for now and focus on the problem in future as it is more global one (similar `as any` is used in `dumbbel`, `varivide`, `xrange`, `polar`, `waterfall` etc.)